packages/rfw/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## NEXT
+## 1.0.9
 
 * Updates minimum supported SDK version to Flutter 3.3/Dart 2.18.
 * Aligns Dart and Flutter SDK constraints.
+* Fix a typo in the API documentation that broke the formatting.
```

Comment:
Nit: Fixes

https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style